content/en/dei/_index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Diversity, Equity, and Inclusion"
+date: 2021-09-17
+draft: false
+---
+
+Healthy communities are built when everyone's voice is heard,
+when their perspective is valued, and when their work is recognized.
+Better technical and social solutions can be found through wide participation.
+We are therefore dedicated to expanding our community in a welcoming and inclusive way,
+with particular focus on increased participation by people who identify with groups that are
+underrepresented in STEM and the open source scientific Python community.
+
+We work closely with NumFOCUS'
+[Diversity & Inclusion in Scientific Computing (DISC) group](https://numfocus.org/programs/diversity-inclusion)
+and the ["Advancing an inclusive culture in the scientific Python ecosystem"]({{< relref "/inclusive_culture_2021_grant.md" >}}) project.
+
```

Comment:
This would be a great place to link to any/all discussion forums etc. For example, something like: `Want to learn more or share your ideas? Join the discussion at <link(s)>`

The DEI category on the discuss forum would be an obvious candidate (if it ends up being decided to use that as a public forum). Whatever the platform ends up being - this seems like a great place to link to it!